datasets/xsum/xsum.py

```diff
@@ -73,39 +96,59 @@ def _info(self):
     def _split_generators(self, dl_manager):
         """Returns SplitGenerators."""
 
-        dl_path = dl_manager.download_and_extract(_URL)
+        data_path = dl_manager.download_and_extract(_URL_DATA)
+        splits_path = dl_manager.download(_URL_SPLITS)
```

Comment:
Could you change this to
```python
files_to_download = {
    "data": _URL_DATA,
    "splits": _URL_SPLITS
}
downloaded_files = dl_manager.download_and_extract(files_to_download)
```
since the download manager is able to do parallel downloads when a dict is passed.
Moreover it should fix the dummy data error in the CI